### PR TITLE
Simplify get credential handler.

### LINKED ISSUE #8: Simplify the handler for getting credentials
This function should be streamlined or eliminated: https://github.com/digitalbazaar/bedrock-credentials-rest/blob/master/lib/services.credentials.js#L237

### PR DESCRIPTION
**Pull request by mattcollier**
Fixes #8.

### CODE REVIEW

**Comment by mattcollier** [comment]
@dlongley Is it correct that this endpoint for URLs like this: `https://issuer.foo.dev:23443/credentials/1.1c.1.1` is accessed by one of three types of identities.  The issuer, the subject, or and administrator.  There is no concept of a public credential on the issuer, is that correct?  And public credentials are handled on the IdP with URL like this `https://idp.foo.dev/credentials?id=https://issuer.foo.dev:23443/credentials/1.1c.1.1` which is handle by a different endpoint.

File: lib/services.credentials.js [hunk 1]
```diff
@@ -208,14 +208,10 @@ bedrock.events.on('bedrock-express.configure.routes', function(app) {
...
-          if(err) {
-            return callback(err);
-          }
-          // FIXME: handle format conversion for partial credentials
-          return _credentialToFormat(
-            record.credential, record.meta, req.query.format, callback);
-        });
+        var credentialId =
+          credentialsRest.createCredentialId(req.params.credential);
+        var actor = (req.user && req.user.identity) ? req.user.identity : null;
```
Comment (dlongley) [inline lib/services.credentials.js]: This isn't correct, it will let anonymous users have full access.

Comment (dlongley) [inline lib/services.credentials.js]: https://github.com/digitalbazaar/bedrock-credentials-rest/blob/718faf6eb1aa9762c16ec4be48658b97cc3d8cda/lib/services.credentials.js#L125

**Comment by dlongley** [comment]
@mattcollier,

> There is no concept of a public credential on the issuer, is that correct?

I don't think we should assume that, no. An issuer could easily be setup to issue public credentials.

**Comment by mattcollier** [comment]
Tested the current implementation with a CURL request: curl -H "Accept: application/json" https://issuer.foo.dev:23443/credentials/1.1.1.1 --insecure

A 404 message is properly returned.

**Comment by dlongley** [comment]
LGTM, merge.

**Pull request merged**